### DISCUSSION TITLE
NestJS AutoMapper returning empty response

### DISCUSSION THREAD

**Discussion by swapab** [nartc/mapper#545] [Q&A]
Thanks for coming up with this plugin. I hope to make my life easier with it.

At the moment, I am unable to make it work.

I have a basic setup [here](https://github.com/swapab/order-service) (with NestJS)

Both API calls are returning empty JSON response
```bash
curl --location --request GET 'http://localhost:3000/users/me'
```

```bash
curl --location --request GET 'http://localhost:3000/users/1'
```

I feel like it is a configuration short coming, however I have followed as much as possible from the [documentation](https://automapperts.netlify.app/docs/nestjs)
Could you please help me make the automapper work it's magic?

Thanks in advance!

**Comment by zgid123** [comment]
it seems your repo is private, you should public it, so everyone can see your repo
